tljh-plasma/tljh_plasma/__init__.py

```diff
@@ -117,12 +117,14 @@ def tljh_custom_jupyterhub_config(c, tljh_config_file=CONFIG_FILE):
     # increase the timeout to be able to pull larger Docker images
     c.PlasmaSpawner.start_timeout = 120
     c.PlasmaSpawner.pull_policy = "Never"
-    c.PlasmaSpawner.remove = True
+    c.PlasmaSpawner.remove = False
     c.PlasmaSpawner.default_url = "/lab"
     # TODO: change back to jupyterhub-singleuser
     c.PlasmaSpawner.cmd = ["/srv/conda/envs/notebook/bin/jupyterhub-singleuser"]
     # set the default cpu and memory limits
     c.PlasmaSpawner.args = ["--ResourceUseDisplay.track_cpu_percent=True"]
+    # explicitely opt-in to enable the custom entrypoint logic
+    c.PlasmaSpawner.run_as_root = True
```

Comment:
This is now opt-in in `12.0`: https://jupyterhub-dockerspawner.readthedocs.io/en/latest/changelog.html#id3

Not sure why we didn't have to set it explicitly before, as it was updated in https://github.com/plasmabio/plasma/pull/193.